rasterio/_err.pyx

```diff
@@ -66,7 +66,7 @@ cdef class GDALErrCtxManager:
         cdef int err_no = CPLGetLastErrorNo()
         cdef const char *msg = CPLGetLastErrorMsg()
         # TODO: warn for err_type 2?
-        if err_type >= 3:
+        if err_type > 0:
```

Comment:
I'm not certain why I chose `3` in the first place, but `0` is what I want for the future.